src/build.sh

```diff
@@ -240,6 +247,33 @@ function full_cleanup()
   cmd_manager "$flag" "$cmd"
 }
 
+# This function is useful for automating the compilation of code
+# with different compilation flags.
+#
+# @flag: Expecting a flag, by default, cmd_manager does not
+# expects flags and always show the command. For more details
+# see the function `cmd_manager` in `src/kwlib.sh`.
+#
+# @env_path: Path of the environment to be used.
+#
+# @parallel_cores: Number of cores to be used during compilation.
+
+function build_cflags()
+{
+  local flag="$1"
+  local env_path="$2"
+  local parallel_cores="$3"
+  local platform_ops="$4"
+  local warnings="$5"
+  local build_flag="${options_values['CFLAGS']}"
+  local cmd
+
+  flag=${flag:-'SILENT'}
+
+  cmd="make -j${parallel_cores} KCFLAGS=${build_flag} ARCH=${platform_ops}${warnings}${env_path}"
+  cmd_manager "$flag" "$cmd"
+}
```

Comment:
This entire function is unnecessary. You need to concatenate KCFLAGS in the line that composes the build command, i.e., this line:
`command="make ${optimizations} ${llvm}ARCH=${platform_ops}${warnings}${output_kbuild_flag}${output_path}"`

Reply:
I made this change like this:
```
command="make ${optimizations} ${llvm}ARCH=${platform_ops}${warnings}"

if [[ -n "$cflags" ]]; then
   command+=" KCFLAGS=${cflags}"
fi
command+="${output_kbuild_flag}${output_path}"
```
However, while reviewing the modification, I considered another option that might be even more efficient. What do you think?
```
command="make ${optimizations} ${llvm}ARCH=${platform_ops}${warnings}${cflags:+ KCFLAGS=${cflags}}${output_kbuild_flag}${output_path}"
```